lib/api/apiUtils/authorization/permissionChecks.js

```diff
@@ -396,6 +399,12 @@ async function validateQuotas(bucket, account, apiNames, apiMethod, inflight, lo
                 bucketName: bucket.getName(),
             });
         }
+        const responseTimeInNs = Number(process.hrtime.bigint() - requestStartTime);
+        monitoring.quotaEvaluationDuration.labels({
+            action: apiMethod,
+            type: 'noCheck',
+            code: 200,
+        }).observe(responseTimeInNs / 1e9);
```

Comment:
I don't see the `noCheck` in the dashboard, and I'm not sure this one provides any value, if there is no quota evaluation it could be ok to not monitor anything to not have unused metric, except if you plan to use it later ?

Reply:
Yes, I agree that there is no point in monitoring "nothing", I will remove it

Reply:
Removed in https://github.com/scality/cloudserver/pull/5556/commits/9facef770aed6b5a13ad72993095f74d20f7eb09